### ISSUE TITLE
A7/A8 4K support . B38- b43 device powered off overnight & reboots 

### ISSUE THREAD

**Issue by Moddingmad** [pwn20wndstuff/Undecimus#635]
After successful use of u0 v3.0.0 b38 . On my iPad mini 2 12.1.2 ( nonce setting successfully ) I left the iPad in jailbroken state . While I wait for the substrate update . But when I awoke this morning and went to use my iPad . I notice it had powered itself off . ( it had not rebooted ) simply turned off. Upon reboot the battery was still on 88% . So it was not a case off battery drain or battery died. Obviously at the moment no tweaks are installed as there is no Cydia or substrate support with Beta 38 . But something is causing a power down .

**Comment by Cryptiiiic** [comment]
@Moddingmad Post diagnostics and the panic-log(bottom most panic-log in logs in settings. Make sure you get the log before jailbreaking again).

**Comment by Moddingmad** [comment]
The u0 Diagnostics ? 
If so I will have to provide this if it happens again. As I have already re-jailbroken.
The panic log from Analytics I will Upload shortly

**Comment by Cryptiiiic** [comment]
Thank you.

**Comment by Moddingmad** [comment]
Updated main post with log file

**Comment by Cryptiiiic** [comment]
@Moddingmad you posted photoanalysisd crash log, not panic-log.

**Comment by Moddingmad** [comment]
Sorry . That is the only log with yesterdays date , No logs that say Panic log.

**Comment by Moddingmad** [comment]
![3ed84e77-d7c0-460a-bca6-a7febedd0140](https://user-images.githubusercontent.com/38045982/53653564-7bdbf980-3c43-11e9-86aa-b58206d2d6f8.jpeg)
This is all I have in relation to 2019 logs

**Comment by Cryptiiiic** [comment]
@Moddingmad Alright see if it happens again tonight. post picture of 2019 crash logs again but see if there are any new ones.

**Comment by Moddingmad** [comment]
Ok . If I see any that say panic logs I will post them , But as of now . I do not see any Panic logs. 
And also if it happens again I will of course upload the u0 Diag log 

Thankyou for responding

**Comment by xMilesManx** [comment]
This exact issue has happened to me over the past three nights. It will reboot overnight consistently. 

My device is an A1522 iPhone 6+ (7,1) 12.1.2 (16C104) Board is N56AP

There are also no panic logs listed in the device analytics. Is there anything else I can do to try to help?

**Comment by Moddingmad** [comment]
Updated to b39 . upon its release . iPad has been jailbroken since release . And so far has not powered itself off overnight. since updating to b39. @Cryptiiiic not sure if you want to close this issue . But all seems stable on B39 so far

**Comment by xMilesManx** [comment]
Can confirm. My iPhone has not rebooted last night either

**Comment by CoolPersonMaam** [comment]
It only causes this issue for some reason if you charge

**Comment by Cryptiiiic** [comment]
@Moddingmad @xMilesManx @CoolPersonMaam Before I close, can you guys test b41 and see if it happens(plug in your phone at night too).

**Comment by Moddingmad** [comment]
Yes I will test this over the next couple of nights. Now that I have loaded the iPad mini 2 with tweaks. I have no loops or random restring issues. So atm all is working 100%. I Will Get back to you with an update if anything strange happens on b41 . @Cryptiiiic

**Comment by Moddingmad** [comment]
@Cryptiiiic 
HI , I did what you asked , I left my iPad on charge last night . whilst jailbroken, on b41 ( iPad mini 2 ) 
When I came down just now (07:40am) I found that the iPad was in NON- jailbroken state ,

I went straight to Analytics. to find No Panic log relating to the date and time .

I went to bed last night at 23:30pm so I looked for anything logged between those times 22:30-07:40

And all I can see are 2 . SystemMemoryReset logs . one happened at 02:00am and another at 03:25am 

And u0 itself tells me to re-Jailbreak . So something is still not happy with b41 . If you need copies of these logs please let me know .

**Comment by xMilesManx** [comment]
@cryptiiic Hi there. 

The issue actually DOES still persist. You are correct in that it only happens when charging. 

I have a bunch of system memory reset logs. I also have a couple of panic logs related to watchdog timer or something. 

I’m on mobile so I cannot upload them easily right away. 

Please let me know if there is anything I can provide to be useful. 

iPhone 6+ 12.1.2

**Comment by Moddingmad** [comment]
I have updated to b42 I will monitor this issue on the latest Beta . And report back any issues I find over the next few nights . when I can next charge overnight as the battery is now full .

**Comment by CoolPersonMaam** [comment]
For the people that have been expriencing this glitch enable Increase Memory Limit. It does drain your battery A LOT however, but it is not as common

**Comment by xMilesManx** [comment]
Update: 

I cleared my analytics logs from settings, charged and went to bed.

System rebooted last night on B42. 

The only log was a system memory reset that had a corresponding time stamp. I can upload to pastebin if needed.

**Comment by Moddingmad** [comment]
Ok so last night I left the iPad off charge for battery to die down . So today I can give a full charge then run battery down again for an overnight charge . Still on b43 . But no powerdown or reboot last night ( unplugged from charger cable ) woke and was still jailbroken with 20% battery . Will keep updating as results come in . 

iPad has been on charge now for about 2hours battery level at 69% no powerdowns or reboots yet .

**Comment by Moddingmad** [comment]
@Cryptiiiic 

I have been testing All day with this pouring off /rebooting issue. as explained in the last comment^^

Update :
9 hours The iPad had been on charge for with no reboot or powering off . So I unplugged the device and just set a video playing on youtube of an aquarium to drain the battery down . So I could perform another full recharge overnight to test for reboot etc again.

30-45 Mins into the video playing . I watched as the iPad display went black then rebooted to the Apple logo. all bt itself ( unplugged from charging cable)
Upon re-jailbreak I have 2 panic logs which I will paste in the next comment .

Hope they will help . 

I will carry on my tests with overnight charging tonight . and provide any panic logs .if device reboots during the night and new panic logs are created

**Comment by Moddingmad** [comment]
Carrying on from the above comment.

last night I connected the charger to my (jailbroken) iPad at 11:30pm device had 16% battery power.
Then went to bed.

When I came down this morning at 08:30AM I found the iPad had rebooted itself again. 
In that 10 hour window whilst being on charge .

Again I headed straight to the Analytics to find the New panic logs. To find None, no new panic logs . 
The only New log showing ( with date and timeframe ) is the same ( SystemMemoryReset ) as I mentioned in previous above comments when this happened the first time. this happened at 02:00am in the morning. 

So I will past this here . hopefully this can point out whats happening .

Also noticed while in Analytics . that I have a vast amount of Jetsamevent Crashes happen every few minutes it seems . even after clearing the logs . they come back almost instantly ( the jetsam event logs)

**Comment by Moddingmad** [comment]
@Cryptiiiic 

So here is a little update on my Ipad / overnight/reboots issue (Saturday night)

So after yesterday's testing and the night before . I found I still had the reboots happening on B43 . And funny as it is. I could charge the ipad during the day (For the same amount of time ) without it rebooting to non jailbreak state . But as soon as I drained the battery and charge it over night for the same amount of hours charging. The reboot happens. ( so is this a overnight issue ) as I have only been able to make this happen overnight on charge. NOT during the day while charging for the same duration of time. (strange) 

Anyway. After these results. I decided to adjust the Jetsam properties. ) based on the - SystemMemoryReset logs- ) 

My ipad Jetsam was set to 55 ( on all 6 jetsam files ) so I lowered all 6 files to 45 . And at 18:30 I set the ipad on charge from 16% battery health .

(Sunday morning)
As it is Sunday I had a bit of a lay in. And woke up at 11am. So the ipad had been on charge for a very long time (15.5 hours). 

When I went to pick it up and touched the screen. It looked like it had rebooted. All my lockscreen tweaks had gone.
So I unlocked the ipad. 
But to my shock I was then presented with a safe mode screen 
which I restarted. And when the device resprung and was still in Jailbroken state.

So i will test again the next couple of days/nights with these jetsam changes . And perform more charges to see if this has fixed the overnight reboots.

**Comment by Moddingmad** [comment]
UPDATE:

So for the past 3/4 days as you can see by the comments above I have been seriously testing and trying different methods to find when the Charging/reboot happens etc. 

The last changes I made were to the jetsam properties. reducing the default Reclaimer limit (55) down to (45)

Then yesterday morning I woke Sunday morning , ( results posted comment above) edited the jetsam properties.
Then performed a full recharge during the day. (total time left on charge 10 hours) result ( no reboot / but at some stage during the charge . The device went into safe mode. ( NO crash reports / no analytic logs . to show why . ( this is with tweaks installed on the iPad ) 

I then discharged the iPad again all the way down to 2% battery health. And again placed the iPad on charge at 23:00pm last night , And then went to bed. 

I woke this morning at 8:00am. First thing I did was look at the iPad. Pressed the home button. The device screen woke up. And again I was presented with a SafeMode screen. No Reboot! 

So my conclusion is. 
Either I have managed to fix the overnight reboot by adjusting the Jetsam properties. 
Or now with those jetsam properties edited. The safe mode that is now happening ,Maybe is preventing the device from overnight/rebooting. Due to being in Safe mode for an amount of time.

My next test will be to remove all tweaks from the device and retest as I have above to see if these issues (reboot/safemode- while on charge ) still happen.  Even tho this issue first started on b38 ( B38 had no Cydia or substrate ) And has been happening ever since for me on the iPad . (A7) I Want to be as thorough as I can be to try to help. 

I will just point out that I am using the exact same u0 beta version on my IOS11.3.1 ipX. running more or less the exact same tweaks. and I do not have any of these issues on the iPhone X at all . No Reboots or Safemode while charging. Only seems to be affecting IOS 12 for me and my devices. 

@Cryptiiiic 
@sbingner 
@pwn20wndstuff 

Some feedback would help. As I have been doing all this testing. And really not getting any Feedback since early comments. 

All I see are new issues about the charging/reboot being opened daily.

**Comment by Moddingmad** [comment]
Ok . so updated the iPad to b44. run the exact same tests . ( removed all tweaks ) iPad is now stock ( but jailbroken) 
And thou I have not come down to a rebooted iPad . I have found it to be in safemode . 

Today when I came down this morning . the device was in safe mode . I restarted ( pressed restart on the safemode error message . and the iPad rebooted . upon reboot . I was unable to re-Jb due to the cert being revoked . (signed with my own Apple ID & reprovision. 

So I have updated again to b45 . and running same tests again . 

After these tests . This will be the last time I post / comment . As it seems this post is now being forgotten about and pushed way down the queue. And other reboot / safemode  issues have been posted .
